cluster/gce/configure-vm.sh

```diff
@@ -968,7 +968,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
yeah, do we expect this CIDR ro be "real"?  Is it in the master Node object?  Does it have a GCE route?  It is conspicuously made-up and un-dynamic.

What happens if I have 2 clusters with 2 masters?  I don't know the history of this logic, but it has a smell to it...